transport/http/outbound.go

```diff
@@ -41,61 +45,61 @@ var (
 	errOutboundNotStarted     = errors.ErrOutboundNotStarted("http.Outbound")
 )
 
-type outboundConfig struct {
-	keepAlive time.Duration
-}
+// NewOutbound is deprecated
+// TODO rename this func NewOutboundFromURL
+func NewOutbound(urlStr string) transport.Outbound {
```

Comment:
The method to indicate something is deprecated is to add a comment in the
following form to the godocs for the deprecated entity.

    Deprecated: ...

So leave the docs for NewOutbound intact and add a Deprecated paragraph to it.

Plus, when deprecating something, indicate what people should do instead.

    // NewOutbound builds a new HTTP outbound that sends requests to the given
    // URL.
    //
    // Deprecated: Do X instead.

See also,
https://blog.golang.org/godoc-documenting-go-code (ctrl-f "deprecated").